lib/assessment.md

```diff
@@ -41,7 +56,20 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+```CSS
+.big{
+    background-color: yellow;
+}
+#contentPane{
+    background-color: blue;
+}
+.important.red{
+    background-color: red;
+}
+div, p{
+    background-color: orange;
+}
```

Comment:
so it should be div> p and not div, p because div>p  takes the direct children from p. When i wrote div , p  does it would grab all divs that that has p in it??